packages/frontend/web/browser/boot.ts

```diff
@@ -135,16 +135,15 @@ const onPolyfilled = (): Promise<void> => {
 };
 
 const run = (): void => {
-    /*
-        We want to run `onPolyfilled` only after polyfill.io has initialised
-        By the time this script runs, if `window.guardian.polyfilled` is true,
-        meaning that polyfill.io has initialised, then we run onPolyfilled(), otherwise
-        we stick it in window.guardian.onPolyfilled to be ran later.
-    */
+    // Expects polyfill to have run
     if (window.guardian.polyfilled) {
```

Comment:
Although, actually we need it for the tests as they currently stand. As otherwise the boot runs on include which breaks them. I did try and remove this but it required a significant refactoring (as the tests all operate at the boot level) so would prefer to defer (haha :grimacing: ) this work to a separate PR.

Reply:
Fair 👍